packages/markdown/parseMarkdown.test.js

```diff
@@ -87,20 +87,14 @@ authors: ['foo', 'bar']
 ---
 title: React Alert component
 components: Alert, AlertTitle
-githubLabel: 'component: alert'
-packageName: '@mui/lab'
-waiAria: https://www.w3.org/TR/wai-aria-practices/#alert
 authors:
   ['foo', 'bar']
```

Comment:
Why removing those lines?

Reply:
The goal is to keep the test more focused on what we try to test, this is unrelated.